sbi/inference/posteriors/base_posterior.py

```diff
@@ -155,30 +156,51 @@ def map(
         num_init_samples: int = 1_000,
         save_best_every: int = 10,
         show_progress_bars: bool = False,
+        force_update: bool = False,
+        warn_about_cached: bool = True,
```

Comment:
I'm a bit hesitant to add two keyword arguments. The second one can be dropped imo.